nel/model/model.py

```diff
@@ -278,7 +279,7 @@ def merge(self, name_entity_iter):
             ne_counts[name][entity] += 1
 
         log.debug('Accumulating %i name->entity counts...', len(name_entity_iter))
-        super(NameProbability, self).merge((name, counts.iteritems()) for name, counts in ne_counts.iteritems())
```

Comment:
Issue here seems to stem from a difference (bug) in the inc_many interfaces for redis and mongo stores. To keep things simple I'll merge as is and make the fix upstream.